src/bots/ama/subcommands/addEvent.js

```diff
@@ -0,0 +1,43 @@
+import client from '../../../client';
```

Comment:
this folder should be subCommands, just to be consistent with our capitalization of that word

Reply:
still need to correct this (also wondering if actions is the better word)